Active Documents/contentEditableDisabled.html

```diff
@@ -7,84 +7,168 @@
     </section>
-    <section data-dfn-for="ElementContentEditable">
+    <section id="contenteditabledisabled">
+        <h2>
+            <code>contenteditabledisabled</code> DOM Content Attribute.
+        </h2>
+        <p>
+            The <code><dfn data-dfn-type="element-attr">contenteditabledisabled</dfn></code> attribute indicates which user agent-supplied editing UI SHOULD be disabled.  When specified, the attribute can contain an [=unordered set of unique space-separated tokens=] that are treated as [=ASCII case-insensitive=].
+        </p>
+        <p>
+            The allowed set of tokens are listed below.  Each token matches the name of an <a data-cite="input-events-1#events-inputevents">Input Type</a>, or is a shortcut that is equivalent to specifying a set tokens.
+        </p>
+        <p>
+            When a token is specified, any user agent-supplied editing UI, that when invoked, would result in a beforeinput event having and inputType that matches one of the tokens in the attribute's value, or matches a token that is represented by a shortcut that is included in the attribute's value, SHOULD be disabled, omitted, or otherwise indicated by the user agent to be not applicable.  The exact mechanism for that indication is beyond the scope of this specification.
+        </p>
+        <h3>
+            <code>contenteditabledisabled</code> Allowed Tokens.
+        </h3>
+        <p>
+            <ul>
+                <li><code>formatjustifycenter</code></li>
+                <li><code>formatjustifyfull</code></li>
+                <li><code>formatjustifyleft</code></li>
+                <li><code>formatjustifyright</code></li>
+                <li><code>formatbackcolor</code></li>
+                <li><code>formatbold</code></li>
+                <li><code>insertlink</code></li>
+                <li><code>formatfontname</code></li>
+                <li><code>formatfontsize</code></li>
+                <li><code>formatfontcolor</code></li>
+                <li><code>formatindent</code></li>
+                <li><code>inserthorizontalrule</code></li>
+                <li><code>insertorderedlist</code></li>
+                <li><code>insertunorderedlist</code></li>
+                <li><code>formatitalic</code></li>
+                <li><code>formatoutdent</code></li>
+                <li><code>formatremove</code></li>
+                <li><code>formatstrikethrough</code></li>
+                <li><code>formatsubscript</code></li>
+                <li><code>formatsuperscript</code></li>
+                <li><code>formatunderline</code></li>
+            </ul>
+        </p>
+        <p class="issue" title="missing fontSize and insertImage">
+            fontSize / insertImage is missing from beforeInput.  Why?  It seems that it should be listed to create a more complete set.
+            <a href="https://github.com/w3c/editing/issues/249">Issue 249</a>
```

Comment:
This should reflect the discussion result. Maybe just a note that those are missing.

Reply:
We discussed this at the meeting and agreed to remove them from both specs for now.